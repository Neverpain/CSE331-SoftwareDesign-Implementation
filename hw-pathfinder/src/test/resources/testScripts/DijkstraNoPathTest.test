# Tests Dijkstra's algorithm on two nodes with no path to
# each other, prints it out

CreateGraph g

AddNode g A
AddNode g B

FindPath g A B