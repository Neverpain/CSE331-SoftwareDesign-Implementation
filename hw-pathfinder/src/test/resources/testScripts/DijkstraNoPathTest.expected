# Tests Dijkstra's algorithm on two nodes with no path to
# each other, prints it out

created graph g

added node A to g
added node B to g

path from A to B:
no path found